Dockerfile

```diff
@@ -1,13 +1,19 @@
 FROM ubuntu:16.04
 
 RUN apt-get update
-RUN apt-get install -y python3-pip
+RUN apt-get install -y wget curl python3-pip
 
-RUN pip3 install --upgrade pip
-RUN pip3 install tensorflow==2.0.0-alpha0 mysql-connector-python pyhive jupyter sqlflow
+# Make python3 as default Python interpreter
```

Comment:
Should we also updated the doc telling user to use python3 and the latest pip version?

Reply:
Absolutely.